src/components/SearchSelectOrText.js

```diff
@@ -68,57 +68,53 @@ export const SearchSelectOrText = (props) => {
 
 
   return (
-    div({ className: 'dropdown select-dropdown', id, name: label }, [
-      a({
-        className: `btn select-btn btn-secondary dropdown-toggle ${errored ? 'errored' : ''}`,
-        role: 'button',
-        id: 'dropdownMenuLink',
-        'data-toggle': 'dropdown',
-        'aria-haspopup': true,
-        'aria-expanded': false,
-        onClick: onOpen,
-        disabled
-      }, [
-        div({
-          style: { width: '100%' }
-        }, [
-          span([currentDisplay]),
-          span({ className: 'caret select-caret caret-margin', style: { right: '2%' } })
-        ])
-      ]),
-      div({
-        className: 'dropdown-menu select-dropdown-menu',
-        onBlur: () => {
-          selectManual(searchTerms.current.value);
-          onBlur && onBlur();
-        }
-      }, [
-        input({
-          type: 'text',
-          placeholder: searchPlaceholder ? searchPlaceholder : 'Search...',
-          className: 'search-bar',
-          onChange:() => handleSearch(searchTerms),
-          onKeyDown: (e) => {
-            if (e.key == 'Enter') {
+    <div className="dropdown select-dropdown" id={id} name={label}>
+      <a
+        className={`btn select-btn btn-secondary dropdown-toggle ${errored ? 'errored' : ''}`}
+        role="button"
+        id="dropdownMenuLink"
+        data-toggle="dropdown"
+        aria-haspopup="true"
+        aria-expanded="false"
+        onClick={onOpen}
+        disabled={disabled}
```

Comment:
It looks like this attribute is not allowed here (and indeed, there are other non-standard attributes scattered across the older hyperscript), but I favor keeping this (and others) here to lessen the potential for regressions.